### PR TITLE
Linux Distribution Installer Support Matrix.

### PR DESCRIPTION
**Pull request by lukehinds**
Outline dist support, plus a few minor fixes / changes to the installer text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,18 +97,30 @@ Options:
...
+Note that CFSSL is required if you want to support revocation. As noted above, do not use the TPM emulator option `-s` in production systems.
+
+#### Installer Distribution coverage
+
+| Distribution  | Versions      | TPM2-Software   |
+| ------------- |:-------------:| -----:          |
+| CentOS        | 7 / 8         | Compiled        |
+| RHEL          | 7 / 8         | Compiled        |
+| Fedora        | 29 / 30 / 31  | Package Install |
+| Ubuntu        | 18.04         | Compiled        |
```
Comment (jetwhiz) [inline README.md]: For Ubuntu, should this be `18 LTS / 19` instead?

Comment (lukehinds) [reply README.md]: Happy to put whatever findings you guys had during testing. 

Should I have put the minor versioning in place too (eg. 19.*04*)

Comment (jetwhiz) [reply README.md]: It's probably fine just to go with `18 LTS / 19`, but I'm not sure how significant the changes between 19.04 and 19.10 will be.

**Comment by lukehinds** [comment]
going to merge this, as the installer patch is in.

**Pull request merged**